libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -4548,3 +4548,34 @@ string YulUtilFunctions::externalCodeFunction()
 		.render();
 	});
 }
+
+std::string YulUtilFunctions::externalFunctionPointersEqualFunction(
+	std::string addressCleanUpFunction,
+	std::string selectorCleanUpFunction
+)
+{
+	std::string const functionName = "externalFunctionPointersEqualFunction";
+	return m_functionCollector.createFunction(functionName, [&]() {
+		return util::Whiskers(R"(
+			function <functionName>(
+				leftAddress,
+				leftSelector,
+				rightAddress,
+				rightSelector
+			) -> result {
+				result := and(
+					eq(
+						<addressCleanUpFunction>(leftAddress), <addressCleanUpFunction>(rightAddress)
+					),
+					eq(
+						<selectorCleanUpFunction>(leftSelector), <selectorCleanUpFunction>(rightSelector)
+					)
+				)
+			}
+		)")
+		("functionName", functionName)
+		("addressCleanUpFunction", addressCleanUpFunction)
+		("selectorCleanUpFunction", selectorCleanUpFunction)
```

Comment:
To clarify: I meant you can just remove the arguments and use this intead:
```suggestion
		("addressCleanUpFunction", *TypeProvider::address())
		("selectorCleanUpFunction", *TypeProvider::uint(32))
```